src/core/mixins/MediaInfo.ts

```diff
@@ -113,28 +118,28 @@ export default class MediaInfo {
     let storyboards;
     let captions;
 
-    if (options.include_thumbnails && player_response.storyboards) {
+    if (manifest_options.include_thumbnails && player_response.storyboards) {
```

Comment:
Should this use `include_thumbnails` from `options` or `manifest_options`?
Looking at `DashOptions` it seems to be the later but then the argument in `options` is misleading

Reply:
The `options` argument type is wrong (will fix it later). Use `manifest_options`.